src/ui/layout/Layout.tsx

```diff
@@ -0,0 +1,22 @@
+import React, { ReactNode } from "react"
+
+import csx from "./Layout.scss"
+
+namespace Layout {
+    export interface Props {
+        className?: string;
+        children: ReactNode;
+        scroll?: boolean;
```

Comment:
scroll is a little bit confusing. This props adds additional height - maybe overgrown props will be better ?

Reply:
done